### ISSUE TITLE
!important missing 

### ISSUE THREAD

**Issue by ij11** [gabrysbiz/css-splitter-maven-plugin#23]
Hey i am working currently with splitter version 1.2.3 and after splitting the css file, on all styles the "!important" is lost which will lead to styling issues ... Do you have an idea? 

Thanks in advance,

Iris

**Comment by Smasherr** [comment]
I also noticed that the fonticons are missing. For example this selector
`.v-ie .base .v-inline-datefield-button:after {
	content: "\200B";
}`
turns into this after splitting:
`.v-ie .base .v-inline-datefield-button:after {
  content: "​";
}`

**Comment by agabrys** [comment]
@ij11 Hi, I confirmed the issue. It will be fixed soon 😄

@Smasherr Hi, please do not add another issue to this topic. You should create a new issue instead. I verified and the problem exists, but it is on [CSS Parser](http://cssparser.sourceforge.net/) side. I even found a similar issue: [#63 Parsing of content: "\A" corrupts the CSS file](https://sourceforge.net/p/cssparser/bugs/63/). I think you should create a new issue in their backlog.

**Comment by Smasherr** [comment]
@agabrys From my point of view it was the same bug. Me and @ij11 we work together and we compared our input and output css files. Both mentioned things were discovered in the same compare session so I thought they would belong together.

**Comment by Smasherr** [comment]
Let's see if I get support on their issue tracker: https://sourceforge.net/p/cssparser/bugs/85/

**Comment by agabrys** [comment]
@Smasherr thank you

@ij11 I think that a snapshot with the fix should be available on next Monday (22-January-2018).

**Comment by agabrys** [comment]
I deployed a 2.0.0-SNAPSHOT version. Please verify it.

**Comment by Smasherr** [comment]
@agabrys Hey, I have tested it yesterday and it worked fine. Although I noticed that the newer versions of plugin remove the newlines - is that the desired behavior? It's not critical, but it makes the resulting stylesheets unreadable.

The character escape sequences has been [resolved](https://sourceforge.net/p/cssparser/bugs/85/#8bea) in the lastest snapshot of CSS Parser, which should be release at the end of this week. Will you please update the dependency before releasing your plugin?

**Comment by agabrys** [comment]
I created a new issue (#26) because we talk about two different problems in one topic.

The plugin adds `\n` if `compress` parameter is set to `false` (default behavior). I checked and for:

    .outlineClr {
    	outline: 0;
    	outline-style: none;
    	outline-width: 0
    }

I received:

    .outlineClr {
      outline: 0;
      outline-style: none;
      outline-width: 0;
    }

The plugin:
* uses two spaces instead of tabulators
* remove "unnecessary" white space characters
* only uses `\n` to do a new line (can be displayed incorrectly by editors which require other sequences, example `notepad.exe` requires `\r\n`, when [Notepad++](https://notepad-plus-plus.org/) displays '\n' correctly).

I'm closing the ticket because the `missing !important` problem has been solved.
